jobs/models.py

```diff
@@ -54,9 +54,15 @@ class Job(PublishFlowModel):
         null=True
     )
     contact_email = models.EmailField(max_length=255)
-    city = models.CharField(max_length=255)
+    cities = models.CharField(max_length=255)
```

Comment:
So cities would be stored as a comma separated string?  If we are allowing for multiple cities, I think it may work out easier for filtering, search etc if it is a one-to-many FK to a cities model.  There are existing apps that deal with cities and may be useful (django-cities and django-cities-light look like the main ones) or get a list of cities to use to populate choices in a simple City model.